modules/cudf/src/node_cudf/table.hpp

```diff
@@ -280,6 +280,8 @@ struct Table : public EnvLocalObjectWrap<Table> {
   static Napi::Value read_orc(Napi::CallbackInfo const& info);
   void write_orc(Napi::CallbackInfo const& info);
 
+  static Napi::Value read_text(Napi::CallbackInfo const& info);
+
```

Comment:
I don't see an implementation for `Table::read_text`, so we shouldn't have a definition in the header here.

```suggestion
```

Reply:
Also left over from the Table method first.